### DISCUSSION TITLE
Question: Vertical separator in Gmail ?

### DISCUSSION THREAD

**Discussion by JombleContact** [InboxSDK/InboxSDK#1093] [General]
**Question(s)**

Hey,

I would like to say thank you at first because you allowed me to get a job (literally) to create an extension in a company.

I just have a problem, I'm using the threadview to create a sidebar and it's working when people are in normal mode (opening the email).

The problem is that when they have divider activated, the sidebar won't open as it's not considered as threadview (see below)

<img width="1387" alt="Capture d’écran 2023-11-28 à 19 09 18" src="https://github.com/InboxSDK/InboxSDK/assets/143142320/481850c7-ffea-4b64-a949-16c156659eea">

May I know what view I should use when user is using this mode ? Thank you once again

**Comment by wegry** [comment]
Hey @JombleContact, two questions for you:

1. What version of `@inboxsdk/core` are you using?
2. Do you have a code sample?

We recently worked through a Gmail DOM change around ThreadViews in #1086. Make sure you're up to date.

**Reply by aiconauser98** [reply]
Hey, I just updated and it's working now.
So sorry for this stupid Issue, and thank you for your fast reply.
